### DISCUSSION TITLE
Why is the value 0 recognized as text instead of an integer when importing a CSV file?

### DISCUSSION THREAD

**Discussion by duomitd** [supabase/supabase#28498] [Questions]
Why is the value 0 recognized as text instead of an integer when importing a CSV file?
[output.csv](https://github.com/user-attachments/files/16556374/output.csv)
id: int8
name:text

**Comment by GaryAustin1** [comment]
There was a bug just fixed in the CSV importer this morning.  Hard refresh to get the fix and see if that does anything for your issue.
